frontend/package.json

```diff
@@ -28,6 +28,8 @@
     "test-suite": "ts-node -O '{\"module\":\"commonjs\"}' ./node_modules/.bin/protractor integration-tests/protractor.conf.ts",
     "test-cypress": "cd packages/integration-tests-cypress && cypress open --env openshift=true",
     "test-cypress-headless": "cd packages/integration-tests-cypress && node --max-old-space-size=4096 ../../node_modules/.bin/cypress run --env openshift=true --browser ${BRIDGE_E2E_BROWSER_NAME:=chrome} --headless",
+    "test-cypress-olm": "cd packages/operator-lifecycle-manager/integration-tests-cypress && cypress open --config-file cypress-olm.json --env openshift=true",
+    "test-cypress-olm-headless": "cd packages/operator-lifecycle-manager/integration-tests-cypress && node --max-old-space-size=4096 ../../../node_modules/.bin/cypress run --config-file cypress-olm.json --env openshift=true --browser ${BRIDGE_E2E_BROWSER_NAME:=chrome} --headless",
```

Comment:
We want this to run with you run `test-cypress` instead of having a separate yarn command. Then we can pass an arg to `test-cypress` if you want to run any particular scenario.

Reply:
We'll want to make sure this runs in CI when `test-gui.sh` is called.

Reply:
@harishsurf, you'll need to follow what `dev-console` has done irt. updating the [yarn scripts](https://github.com/openshift/console/pull/6776/files#diff-0a6694dd0d6b93a2aa14f32e3d873b4c).  
You'll also need to include the Reporters [here](https://github.com/openshift/console/pull/6776/files#diff-6f2ef64ef74ee3c4091b2665adbb373dR20) and [here](https://github.com/openshift/console/pull/6776/files#diff-c42dd9cc6e3aad4af77fe8d29faad3ff).  As well as [copying](https://github.com/openshift/console/pull/6776/files#diff-6f2ef64ef74ee3c4091b2665adbb373dR24) cypress screenshots and recordings to `frontend/gui_test_screenshots`.

Reply:
@harishsurf, pls follow how `dev-console` has integrated with our CI tests.
https://github.com/openshift/console/pull/6776/files#diff-da6498268e99511d9ba0df3c13e439d10556a812881c9d03955b2ef7c6c1c655R30

Reply:
Couldn't we just write a simple automation script that searches for a cypress test folder in each package, and runs `cypress open` in that dir if it exists?

Reply:
This might help clarify what we're after:  https://issues.redhat.com/browse/CONSOLE-2426